src/store/setting.ts

```diff
@@ -267,6 +268,52 @@ export const settingStore: VoiceVoxStoreOptions<
       window.electron.setSplitterPosition(splitterPosition);
       commit("SET_SPLITTER_POSITION", { splitterPosition });
     },
+
+    /**
+     * CPU/GPUモードを切り替えようとする。
+     * GPUモードでエンジン起動に失敗した場合はCPUモードに戻す。
+     */
+    CHANGE_USE_GPU: createUILockAction(
+      async ({ state, dispatch }, { useGpu }) => {
+        if (state.useGpu === useGpu) return;
+
+        const isAvailableGPUMode = await window.electron.isAvailableGPUMode();
+
+        // 対応するGPUがない場合に変更を続行するか問う
+        if (useGpu && !isAvailableGPUMode) {
+          const result = await window.electron.showQuestionDialog({
+            type: "warning",
+            title: "対応するGPUデバイスが見つかりません",
+            message:
+              "GPUモードの利用には、メモリが3GB以上あるNVIDIA製GPUが必要です。<br />" +
```

Comment:
```suggestion
              "GPUモードの利用には、メモリが3GB以上あるNVIDIA製GPUが必要です。\n" +
```

![image](https://user-images.githubusercontent.com/34832037/171031598-a38eea06-a08e-46a3-b379-2b87bafa9147.png)
Windowsのダイアログを使うようにしたことで、HTMLの改行が使えなくなっているので、このような表記になっています。
普通に改行に置き換えることで解決できます。

Reply:
おお！！　ありがとうございます！！